src/cf-layout.less

```diff
@@ -1203,16 +1271,28 @@
         });
     }
 
+    &__flush {
+        margin-top: 0 !important;
+        margin-bottom: 0 !important;
+        margin-right: -(@grid_gutter-width / 2);
+        margin-left: -(@grid_gutter-width / 2);
+        .respond-to-min(600px, {
```

Comment:
Trivial: Please include blank lines between groups of properties, nested blocks, and responsive mixin blocks.